data/logs_model/logs_producer/splunk_logs_producer.py

```diff
@@ -62,6 +65,12 @@ def __init__(
             raise Exception(
                 "Connection to Splunk refused, check if Splunk instance is available: %s" % e
             )
+        except Exception as e:
+            strict_logging_disabled = config.app_config.get("ALLOW_WITHOUT_STRICT_LOGGING")
+            if strict_logging_disabled:
+                logger.exception("log_action failed: %s", e)
```

Comment:
technically this isn't a `log_action`, we should output something like `unable to create index`